proto/frequenz/api/common/v1/pagination.proto

```diff
@@ -27,3 +27,42 @@ message PaginationInfo {
   // Token for retrieving the next set of results.
   string next_page_token = 2;
 }
+
+
+// !!! example "Example of a service that uses pagination."
```

Comment:
I think this will not be rendered on the website though. I think it's not converted to markdown by protoc-gen-doc. Any ideas on where to put it so it is rendered? 
Also I can't render the website, installed the dependencies first and the used `mkdocs-serve`. But it's not working, also no on the main branch. Does it work for you guys?

Reply:
I think for now you could put code examples in backticks, like this:

```
this is a code example
```

Reply:
>Also I can't render the website, installed the dependencies first and the used `mkdocs-serve`. But it's not working, also no on the main branch. Does it work for you?

It does work for me, but I have found issues with it. Happy to talk about this and sort it out on your end in a call.

Reply:
I'd suggest adding the example to the doc for the `message`.